Image/image.css

```diff
@@ -0,0 +1,49 @@
+@import '../style.css';
+
+.responsive-img-container{
+    display: flex;
+    flex-wrap: wrap;
+    width:100%;
+    height: max-content;
+    justify-content: space-evenly;
+    align-items: baseline;
+    margin:10px;
+}
```

Comment:
Add object-fit property in images it would come handy later

Reply:
added